mods/rv/AUTHORS.RV

```diff
@@ -341,6 +341,9 @@ files each credit refers to can be found in
    > Apocalypse Tank Turret
    > Havoc Attack Helicopter
 
+ * SxWxSGame(spiritwitchspiritgame@gmail.com)
+   > New prism explosion
```

Comment:
Just call it Prism Explosion. Also you are still missing this in mods/rv/bits/AUTHORS.txt.